assets/src/modules/LocateByLayer.js

```diff
@@ -291,10 +291,15 @@ export default class LocateByLayer {
             for (var j=0, featuresLen=features.length; j<featuresLen; j++) {
                 var featElement = features[j];
                 locate.features[featElement.id.toString()] = featElement;
-                if ( !('filterFieldName' in locate) )
+                if ( !('filterFieldName' in locate) ) {
+                    // add option
                     options += '<option value="' + featElement.id + '">'
-                        + DOMPurify.sanitize(featElement.properties[locate.fieldName].toString())
-                        + '</option>';
+                    // DOMPurify.sanitize needs a string
+                    options += featElement.properties[locate.fieldName] !== null ?
+                        DOMPurify.sanitize(featElement.properties[locate.fieldName].toString()) :
+                        'NULL'; // Default label for null value
```

Comment:
@nboisteault @Gustry @mdouchin : Empty string or `NULL` when the value is `null`.

Reply:
To be iso as before DOMPurify, we have to display `null`. I will change it.

For an other version, we could add an option to do not add `null` to the list with some log. https://github.com/3liz/lizmap-web-client/issues/3758